pset-functions.js

```diff
@@ -30,7 +30,19 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 1:')
 
 // Add your code below this line
-
+function blackJack(playerCardScore, dealerCardScore) {
+  if (playerCardScore <= 21 && dealerCardScore <= 21) {
+    return Math.max(playerCardScore, dealerCardScore)
+  } else if (playerCardScore <= 21) {
+    return playerCardScore
+  } else if (dealerCardScore <= 21) {
+    return dealerCardScore
+  } else {
+    return 0
+  }
+}
+
+console.log(blackJack(19, 21))
 // Add your code above this line
```

Comment:
@OcielC Good Job here on this solution. Using the conditionals to check for the outcomes for the game and utilizing the Math.max method was a good choice for this use case.

Take a look at this solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L30)